src/ImageDraw.jl

```diff
@@ -51,4 +52,7 @@ export
 	#Cross
 	Cross
 
+	#Polygon fill
+	BoundaryFill
```

Comment:
Without the comma, `BoundaryFill` is not exported.

```suggestion
      Cross,

      #Polygon fill
      BoundaryFill
```